common/constants/constants.go

```diff
@@ -71,12 +71,16 @@ const (
 	// Rule types
 	GlobalDataTypeChange = "global_datatype_change"
 	AddIndex             = "add_index"
+
 	//bulk migration type
 	BULK_MIGRATION = "bulk"
 	//dataflow migration type
 	DATAFLOW_MIGRATION = "dataflow"
 	//DMS migration type
-	DMS_MIGRATION       = "dms"
+	DMS_MIGRATION = "dms"
+	//dataproc migration type
+	DATAPROC_MIGRATION = "dproc"
```

Comment:
nit: better to use the whole name here

Reply:
Done,
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/520/commits/c187c901898bd2f01b3f9aec41e787d93cb7aa12